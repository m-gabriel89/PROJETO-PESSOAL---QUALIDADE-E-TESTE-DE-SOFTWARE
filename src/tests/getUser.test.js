// getUser.test.js
const getUser = require('./getUser');

test('Snapshot do usuÃ¡rio', () => {
    expect(getUser()).toMatchSnapshot();
});
