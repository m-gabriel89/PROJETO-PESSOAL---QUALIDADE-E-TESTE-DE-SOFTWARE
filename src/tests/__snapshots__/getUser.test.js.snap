// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot do usu√°rio 1`] = `
{
  "id": 1,
  "name": "Alice",
}
`;
